src/hooks/useInstanceSettings.jsx

```diff
@@ -12,7 +12,7 @@ const useInstanceSettings = client => {
         const response = await client.query(
           client.all('io.cozy.settings').getById('instance')
         )
-        setSettings(get(response, 'attributes'), {})
+        setSettings(get(response, 'data.attributes'), {})
```

Comment:
Just to be sure, cozyClient was updated prior to this bugfix right ?

Reply:
Yes. I updated cozy-client (for other reasons) but forgot to change this line.